moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1184,40 +1184,40 @@ class MoveGroup::MoveGroupImpl
 }
 }
 
-moveit::planning_interface::MoveGroup::MoveGroup(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
+moveit::planning_interface::MoveGroupInterface::MoveGroupInterface(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
```

Comment:
im still curious about this discussion of namespaces, but i think this PR is ready to merge